products/notepad++.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Notepad++
+category: app
+iconSlug: Notepad++
+permalink: /notepadpp
+changelogTemplate: "https://notepad-plus-plus.org/downloads/v__LATEST__/"
+
+auto:
+  methods:
+  -   git: https://github.com/notepad-plus-plus/notepad-plus-plus
+
+identifiers:
+-  repology: notepad%2B%2B
+
+releases:
+-   releaseCycle: "8.7"
+    releaseDate: 2024-08-28
+    eol: false
+    latest: '8.7.8'
+    latestReleaseDate: 2025-03-08
+
+---
+
+> [Notepad++](https://npp-user-manual.org/docs/getting-started/) is a free (as in “free speech”
+> and also as in “free beer”) source code editor and Notepad replacement that supports several 
+> languages. Running in the MS Windows environment, its use is governed by GNU General Public License.
+
+## General Support
+
+[Statement regarding support](https://community.notepad-plus-plus.org/post/88471): 
+There will never be a “v8.4.6 bugfix” version once a version after v8.4.6 has been released. Bug fixes always go in the most recent release.
```

Comment:
https://community.notepad-plus-plus.org/post/88471 is a great link. We could say something like that to be more explicit.

> Notepad++ does not have a documented release policy. According to [this comment](https://community.notepad-plus-plus.org/post/88471) only the latest release is maintained.

This will also have to be confirmed using 3-4 of the latest releases.

Reply:
Applied in https://github.com/endoflife-date/endoflife.date/pull/7048/commits/fad10327be475e8c8aea8acaf4b8ee5acef0f52a.